### PR TITLE
NVDA screen reader alerts

### PR DESCRIPTION
**Pull request by DDoucetHC**
added a delay to reload the message and moved the role=alert around

### CODE REVIEW

File: angular.json
Comment (dmiraflorhc) [inline angular.json:1]: Not sure what this is for

Comment (DDoucetHC) [reply angular.json:1]: not sure I didn't add it maybe one of the scripts I ran added it? I will remove and make sure it still works

**Review by dmiraflorhc — APPROVED** [review]

**Review by linghanHC — COMMENTED** [review]
the div already has the ngIf condition, not sure why creating another flag for the span

**Comment by AMckayHC** [comment]
If you remove the ngIf on the div does it still work? Maybe NVDIA handles the span better than div for some reason? Just wondering in case Jaws will repeat it twice (div and span)? Unfortunately we don't have access to Jaws to try.

**Comment by DDoucetHC** [comment]
I've changed the div / span around to only contain one ngif

**Comment by linghanHC** [comment]
target branch should be https://github.com/hres/AngularApps/tree/angular17

**Comment by AMckayHC** [comment]
We will wait to merge these changes once Diana has merged her AI changes first.

**Pull request closed without merging**